### PR TITLE
fix: adding missing color to saved search preview name

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR solves [APPL-605]

The issue came from the fact that we were missing some colors in the list in MP. I added the missing color as well as made this less defensive to avoid breaking the UI.

https://github.com/artsy/metaphysics/assets/11945712/63af94fe-13f5-4450-ac29-a986b3b5a504

[APPL-605]: https://artsyproduct.atlassian.net/browse/APPL-605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lib/colors.ts [hunk 1]
```diff
@@ -1,138 +1,149 @@
...
+  foregroundColor: string
+  name: string
+  value: string
+}[] = [
+  {
+    backgroundColor: "#7B5927",
+    foregroundColor: "#fff",
+    name: "Brown",
+    value: "brown",
+  },
```
Comment (anandaroop) [inline src/lib/colors.ts:94]: 👍🏽

**Review by anandaroop — APPROVED** [review]
Sounds reasonable, though it's interesting it took this long to surface!

**Comment by MounirDhahri** [comment]
will address your comments @dblandin

**Review by dblandin — APPROVED** [review]
Perfect. Thank you!

**Pull request merged**